### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by jxchen01**
update readme with

- description
- installation instruction
- quick start guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
... Allen Cell Structure Segmenter plugin for napari provides an intuitive graphical user interface to access the powerful segmentation capabilities of an open source 3D segmentation software package developed and maintained by the Allen Institute for Cell Science (classic workflows only with v1.0). ​The Allen Cell Structure Segmenter is a Python-based open source toolkit developed at the Allen Institute for Cell Science for 3D segmentation of intracellular structures in fluorescence microscope images. This toolkit brings together classic image segmentation and iterative deep learning workflows first to generate initial high-quality 3D intracellular structure segmentations and then to easily curate these results to generate the ground truths for building robust and accurate deep learning models. The toolkit takes advantage of the high replicate 3D live cell image data collected at the Allen Institute for Cell Science of over 30 endogenous fluorescently tagged human induced pluripotent stem cell (hiPSC) lines. Each cell line represents a different intracellular structure with one or more distinct localization patterns within undifferentiated hiPS cells and hiPSC-derived cardiomyocytes.
```
Comment (thao-do) [inline README.md:11]: `The Allen Cell Structure Segmenter plugin...` should be `The Allen Cell & Structure Segmenter plugin..`

File: README.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
-# napari-allencell-segmenter
+# napari-aicssegmentation
```
Comment (thao-do) [inline README.md:1]: I thought the previous name (napari-allencell-segmenter) was correct?

File: README.md [hunk 3]
```diff
@@ -24,9 +25,32 @@ https://napari.org/docs/plugins/index.html
 
 ## Installation
 
-You can install `napari-allencell-segmenter` via [pip]:
+### Optino 1 (recommended):
+
+After you installed the lastest version of Napari, you can go to "Plugins" --> "Install/Uninstall Package(s)". Then, you will be able to see all available Napari plugins and you can find us by name "napari-aicssegmentation". Just click the "install" button to install the Segmenter plugin.
```
Comment (thao-do) [inline README.md:30]: `napari` should be all lowercase.

**Comment by Aerendel** [comment]
Please use https://github.com/AllenCell/napari-allencell-segmenter/pull/112

File: README.md [hunk 4]
```diff
@@ -24,9 +25,32 @@ https://napari.org/docs/plugins/index.html
 
 ## Installation
 
-You can install `napari-allencell-segmenter` via [pip]:
+### Optino 1 (recommended):
```
Comment (thao-do) [inline README.md:28]: typo: `Option`

**Pull request closed without merging**